### PR TITLE
initial version of multus blog post

### PR DESCRIPTION
**Pull request by yuvalif**

### CODE REVIEW

File: _posts/2018-09-12-attaching-to-multiple-networks.markdown [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+description: This post describes how to connect a Virtual Machine to more than one network using the Multus CNI.
+navbar_active: Blogs
+pub-date: September 12
+pub-year: 2018
+category: news
+comments: true
+---
+
+# Introduction
+Virtual Machines often need multiple interfaces connected to different networks. This could be because the application running on it expect to be connected to different interfaces (e.g. a virtual router running on the VM); because the VM need L2 connectivity to a network not managed by k8s (e.g. allow for PXE booting); or any other reason - which I'll be happy to hear about in the comment section!
```
Comment (xsgordon) [inline _posts/2018-09-12-attaching-to-multiple-networks.markdown]: I would also be explicit that many *existing* VMs have this expectation.

File: _posts/2018-09-12-attaching-to-multiple-networks.markdown [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+In KubeVirt, as nicely explained in this [blog post](http://kubevirt.io/2018/KubeVirt-Network-Deep-Dive.html), there is already a mechanism to take an interface from the pod and move it into the Virtual Machine. However, k8s allows for a single network plugin to be used in a cluster (across all pods), and provide one interface for each pod.
```
Comment (xsgordon) [inline _posts/2018-09-12-attaching-to-multiple-networks.markdown]: This forces us to choose between having pod network connectivity and any other network connectivity for the Pod and, in the context of KubeVirt, the Virtual Machine within.

File: _posts/2018-09-12-attaching-to-multiple-networks.markdown [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+To overcome this limitation, we use [Multus](https://github.com/intel/multus-cni), which is a "meta" CNI (Container Network Interface), allowing multiple CNIs to coexist, and allow for a pod to use the right ones for its networking needs.
+
+# How Does it Work for Pods?
+The magic is done via a new CRD (Custom Resource Definition) called ```NetworkAttachmentDefinition``` introduced by the Multus project, and adopted by the k8s community as the [de-facto standard](https://docs.google.com/document/d/1Ny03h6IDVy_e_vmElOqR7UdTPAG_RNydhVE1Kx54kFQ/edit#heading=h.hylsbqoj5fxd) for attaching pods to one or more networks. These network definition contains a field called ```type``` which indicates the name of the actual CNI that provide the network, and different configuration payloads which the Multus CNI is passing to the actual CNI. For example, the following network definition:
```
Comment (xsgordon) [inline _posts/2018-09-12-attaching-to-multiple-networks.markdown]: s/k8s/Kubernetes

No need to abbreviate here.

**Comment by scollier** [comment]
Agree, great first pass.

**Comment by tripledes** [comment]
[(5) [epic-supportability] L2 Networking Blog Post](https://trello.com/c/dtuBaPcI/64-5-epic-supportability-l2-networking-blog-post)

**Comment by scollier** [comment]
@yuvalif looks like there's a broken link.  Please fix and we'll merge it in.

**Comment by yuvalif** [comment]
@scollier I have clicked all links in the post, and they all work. according to the error log, the broken link is to the blog post itself:
```External link https://www.kubevirt.io//2018/attaching-to-multiple-networks.html failed: 404```
Not sure how to fix that?

**Comment by yuvalif** [comment]
[multus blog post](https://trello.com/c/vLk3tGAD/204-multus-blog-post)

**Comment by karmab** [comment]
let's rebase first

File: _posts/2018-09-12-attaching-to-multiple-networks.markdown [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
+      "type": "ovs",
+      "bridge": "blue100",
+      "vlan": 100
+    }'
+```
+- More information could be found in the [OVS CNI documentation](https://github.com/kubevirt/ovs-cni/blob/master/docs/deployment-on-arbitrary-cluster.md)
+## Deploy a Virtual Machine with 2 Interfaces
```
Comment (karmab) [inline _posts/2018-09-12-attaching-to-multiple-networks.markdown]: maybe a space here?

File: _posts/2018-09-12-attaching-to-multiple-networks.markdown [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+      "type": "ovs",
+      "bridge": "blue"
+    }'
+```
+- To use as specific port/vlan from that bridge, you should first create one:
+```bash
+ovs-vsctl add-br blue1 blue 100
+```
+- Then, define its ```NetworkAttachmentDefinition``` CRD:
+```yaml
```
Comment (karmab) [inline _posts/2018-09-12-attaching-to-multiple-networks.markdown]: one more space, this doesnt get rendered

**Comment by iranzo** [comment]
https://github.com/codeship/documentation/pull/1344 related ?

https://github.com/gjtorikian/html-proofer/issues/170

and https://github.com/gjtorikian/html-proofer/issues/170#issuecomment-384980061

**Comment by iranzo** [comment]
@karmab seems that #143 fixed it :-)

**Pull request merged**